kadmin/load.c

```diff
@@ -607,6 +608,9 @@ doit(const char *filename, int mergep)
 	}
     }
     free(line);
+    ret2 = db->hdb_set_sync(context, db, 1);
+    if (ret2)
+        krb5_err(context, 1, ret, "failed to sync the HDB");
```

Comment:
Can you coalesce these as done in master, i.e.

```
    ret2 = db->hdb_set_sync(context, db, 1);
    if (ret2) {
        krb5_err(context, 1, ret2, "failed to sync the HDB");
        ret = ret2;
    }
```

Reply:
They shouldn't have been coalesced.  The hdb_set_sync() call needed to be inserted after the `if (ret2) ret = ret2;` statement.